frontend/apps/reader/modules/readergesture.lua

```diff
@@ -42,17 +51,36 @@ function ReaderGesture:addToMainMenu(menu_items)
     menu_items.gesture = {
         text = _("Gesture manager"),
         sub_item_table = {
+            {
+                text = _("Enable multiswipes"),
+                checked_func = function() return self.multiswipes_enabled end,
+                callback = function()
+                    G_reader_settings:saveSetting("multiswipes_enabled", not self.multiswipes_enabled)
+                    self.multiswipes_enabled = G_reader_settings:isTrue("multiswipes_enabled")
+                end,
+                hold_callback = function()
+                    UIManager:show(InfoMessage:new{
+                        text = multiswipes_info_text
+                    })
```

Comment:
I think `help_text=multiswipes_info_text` just does all that :)

Reply:
Drat, I forgot about that. :-)